app-emulation/waydroid/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=waydroid
+PKGSEC=utils
+PKGDEP="systemd nftables pygobject-3 gtk-3 dbus-python dnsmasq lxc iproute2 \
+        python-3 gbinder-python pyclip"
+PKGDES="An Android container manager"
+
+ABHOST=noarch
+
+# It's just python
```

Comment:
Then this should not be needed. Autobuild will not attempt to split out -dbg if the package is marked as noarch.